src/types/Todo.ts

```diff
@@ -0,0 +1,5 @@
+export type Todo = {
```

Comment:
```suggestion
export interface Todo = {
```
use interface prefer